source/index.html.md

```diff
@@ -359,6 +359,8 @@ eclair-cli updaterelayfee \
 }
 ```
 
+Returned `feeBase` is in msat.
```

Comment:
Not sure this is necessary, a couple lines below it's obvious that it's `msat`.

Reply:
True, shouldn't hurt though, right?

Reply:
Agreed, let's keep it if you think it's useful.